xbmc/addons/kodi-addon-dev-kit/include/kodi/libKODI_guilib.h

```diff
@@ -170,10 +170,10 @@ typedef struct CB_GUILib
 
 
 /* current ADDONGUI API version */
-#define KODI_GUILIB_API_VERSION "5.11.0"
+#define KODI_GUILIB_API_VERSION ADDON_GLOBAL_VERSION_GUI
 
 /* min. ADDONGUI API version */
-#define KODI_GUILIB_MIN_API_VERSION "5.10.0"
+#define KODI_GUILIB_MIN_API_VERSION ADDON_GLOBAL_VERSION_GUI
```

Comment:
```
# cat /usr/share/kodi/addons/kodi.guilib/addon.xml 
<?xml version="1.0" encoding="UTF-8"?>
<addon id="kodi.guilib" version="#define KODI_GUILIB_API_VERSION ADDON_GLOBAL_VERSION_GUI" provider-name="Team-Kodi">
  <backwards-compatibility abi="#define KODI_GUILIB_MIN_API_VERSION ADDON_GLOBAL_VERSION_GUI"/>
  <requires>
    <import addon="xbmc.core" version="0.1.0"/>
  </requires>
</addon>
```

any addon that depends on guilib is probably broken now. 
https://github.com/xbmc/xbmc/blob/master/cmake/scripts/common/Macros.cmake#L632-L634 should also be changed